static/app/views/dashboards/manage/dashboardTable.spec.tsx

```diff
@@ -258,9 +265,16 @@ describe('Dashboards - DashboardTable', function () {
         onDashboardsChange={dashboardUpdateMock}
       />
     );
+    renderGlobalModal();
 
     await userEvent.click(screen.getAllByTestId('dashboard-duplicate')[1]);
 
+    expect(createMock).not.toHaveBeenCalled();
```

Comment:
```suggestion
    expect(postMock).not.toHaveBeenCalled();
```
From the test is looks like this should be `postMock`, that's what's tested just below 👀

Reply:
oh yesss forgot to change that, good catch!